.coveragerc

```diff
@@ -0,0 +1,13 @@
+[run]
+branch = True
+source = nilearn
+parallel = True
```

Comment:
If its unused then we should remove it. Cruft like this would make it harder to read the existing code, esp for new devs.
OTOH pytest-cov is not providing us any overt benefits I had added that before I realized the site-packages thing. We can remove it and if it doesn't break anything we can leave it out.

Reply:
> OTOH pytest-cov is not providing us any overt benefit

sorry I'm not sure what you mean? pytest-code is needed to get coverage reports right?

Reply:
pleas try removing `source = nilearn` since it is both a path and a package name it seems that can interfere with coverage collection

Reply:
> sorry I'm not sure what you mean? pytest-code is needed to get coverage reports right?

I'm tallking about `pytest-cov` plugin that this PR is currently installing.

Reply:
AFAIK it is required to get a .coverage from running pytest isn't that the case?